docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-offer-feature-integration.md

```diff
@@ -98,28 +99,60 @@ Verify that the following changes have been implemented by checking your databas
 | spy_product_offer_store                      | table  | created |
 | spy_product_concrete_product_offers_storage  | table  | created |
 | spy_product_offer_storage                    | table  | created |
-| spy_sales_order_item.product_offer_reference | column | created |
```

Comment:
is it covered in the corresponding IG?

Reply:
it is in `marketplace-order-management-feature-integration.md`

Reply:
`spy_sales_order_item.product_offer_reference` is not presented in validation section in  `marketplace-order-management-feature-integration.md`